xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/transactions/Wrappers.java

```diff
@@ -231,4 +232,76 @@ public String toString() {
     }
 
   }
+
+  /**
+   * A wrapped {@link String} containing the NFT Id.
+   */
+  @Value.Immutable
+  @Wrapped
+  @JsonSerialize(as = NfTokenId.class)
+  @JsonDeserialize(as = NfTokenId.class)
+  abstract static class _NfTokenId extends Wrapper<String> implements Serializable {
+
+    @Override
+    public String toString() {
+      return this.value();
+    }
+
+    /**
+     * Validates that a NfTokenId value's length is equal to 64 characters.
+     */
+    @Value.Check
+    public void validateLength() {
+      Preconditions.checkArgument(this.value().length() == 64, "TokenId must be 64 characters long.");
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+      if (obj != null && obj instanceof NfTokenId) {
+        String otherValue = ((NfTokenId) obj).value();
+        if (otherValue != null) {
+          return otherValue.toUpperCase(Locale.ENGLISH).equals(value().toUpperCase(Locale.ENGLISH));
+        }
+      }
+      return false;
+    }
+  }
+
+  /**
+   * A wrapped {@link com.google.common.primitives.UnsignedInteger} containing the TransferFee.
+   */
+  @Value.Immutable
+  @Wrapped
+  @JsonSerialize(as = TransferFee.class)
+  @JsonDeserialize(as = TransferFee.class)
+  abstract static class _TransferFee extends Wrapper<UnsignedInteger> implements Serializable {
+
+    /**
+     * Construct {@link TransferFee} as a percentage value.
+     *
+     * @param percent of type {@link BigDecimal}
+     * @return {@link TransferFee}
+     */
+    static TransferFee ofPercent(BigDecimal percent) {
+      return TransferFee.of(UnsignedInteger.valueOf(percent.scaleByPowerOfTen(2).toBigIntegerExact()));
+    }
+
+
+    /**
+     * Validates that a NfTokenId value's length is equal to 64 characters.
+     */
+    @Value.Check
+    public void validateBounds() {
+      Preconditions.checkArgument(
+        FluentCompareTo.is(value()).lessThanOrEqualTo(UnsignedInteger.valueOf(9999)) &&
+          FluentCompareTo.is(value()).greaterThanEqualTo(UnsignedInteger.valueOf(0)),
+        "TransferFee should be in the range 0 to 9999.");
+    }
+
+    public boolean equals(UnsignedInteger obj) {
+      return FluentCompareTo.is(value()).equalTo(obj);
+    }
```

Comment:
I think this is unnecessary. Developers should not be comparing `TransferFee`s with `UnsignedInteger`s. If you want to compare a `TransferFee` to something, it has to be another `TransferFee`, otherwise you lose the benefits of using `TransferFee` in the first place. If you want to compare a `TransferFee` to an `UnsignedInteger`, just do `transferFee.value().equals(unsignedInt)`.